linux/deploy_vm/rebuild_pardus_unattend_install_iso.yml

```diff
@@ -18,6 +18,18 @@
       - 'isolinux/install.cfg'
       - 'md5sum.txt'
 
+- name: "Check file /install/gtk/vmlinuz exists in file boot/grub/grub.cfg"
+  ansible.builtin.command: "grep '\/install\/gtk\/vmlinuz' {{ src_iso_file_dir }}/grub.cfg"
+  register: kernel_initrd_dir_result
+  ignore_errors: true
+  delegate_to: "{{ vm_guest_ip }}"
+
+- name: "Set fact of the kernel and initrd file directory"
+  ansible.builtin.set_fact:
+    kernel_initrd_dir: |-
+      {%- if kernel_initrd_dir_result.failed is defined and not kernel_initrd_dir_result.failed -%}/install/
+      {%- else -%}/install/gtk/{%- endif -%}
```

Comment:
when grep command didn't fail, it represents /install/gtk exists, then kernel_initrd_dir should be /install/gtk